.repos

```diff
@@ -0,0 +1 @@
+/Users/jonaprieto/.local/share/haskell-gtd-nl/repos
```

Comment:
Remove this file - it must have been staged / committed before the `.repos` entry in the `.gitignore` file was added.